pkgs/applications/video/mpv/scripts/memo.nix

```diff
@@ -0,0 +1,34 @@
+{ fetchFromGitHub
+, buildLua
+, lib
+, unstableGitUpdater
+}:
+
+buildLua {
+  pname = "memo";
+  version = "unstable-2023-10-31";
+
+  src = fetchFromGitHub {
+    owner = "po5";
+    repo = "memo";
+    rev = "6f2b036ef860e130ea584657389e0a7ceb4d954f";
+    sha256 = "03rk1f97jblnajgwr36qz7hq1qlxrgkywc0gnq0wwg1vxiga9j4v";
```

Comment:
```suggestion
    hash = "...";
```

Reply:
fixed in [b0feaf53aefa0cfce056767e51ea658254b0fb33](https://github.com/NixOS/nixpkgs/pull/288105/commits/b0feaf53aefa0cfce056767e51ea658254b0fb33)